### PR TITLE
Problems solved on 10/1

### PR DESCRIPTION
**Pull request by HidemichiShimura**
These are problems on LeetCode which were solved on 10/1.

### CODE REVIEW

File: 1304.find-n-unique-integers-sum-up-to-zero.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+/*
+ * @param {number} n
+ * @return {number[]}
+ */
+
+var sumZero = function(n) {
+    const uniqueIntArr = [];
+    for (let i = 0; i < Math.floor(n / 2); i++) {
+        const randomNum = i + 5;
```
Comment (ryonryon) [inline 1304.find-n-unique-integers-sum-up-to-zero.js:13]: is it random?

**Review by ryonryon — APPROVED** [review]

**Review by yuya-minamide — APPROVED** [review]
I checked them.

File: 1304.find-n-unique-integers-sum-up-to-zero.js (hunk 1 above)
Comment (HidemichiShimura) [reply 1304.find-n-unique-integers-sum-up-to-zero.js]: I changed "randomNum" to "uniqueNum"

**Comment by HidemichiShimura** [comment]
Thank you for reviewing.

**Pull request merged**